### PR TITLE
Integrating Web Application Changes into Main Branch (Issue #101)

### PR DESCRIPTION
**Pull request by addie-p**
Tested the complete compatibility/integration of all features and components in moot branch (integrate_main). Creating a merge request to add all changes to main branch.

### CODE REVIEW

**Review by SandraV255 — APPROVED** [review]
Reviewed, changes look good to push to main branch. All set!

**Comment by evc29** [comment]
Reviewed all code, the website and components works properly and it all looks great to push to main!

**Review by evc29 — COMMENTED** [review]
Reviewed all code, the website and components works properly and it all looks great to push to main!

**Review by evc29 — APPROVED** [review]

**Review by cding888 — APPROVED** [review]
Reviewed all files and everything seems to align with our project goals. It's impressive seeing everyones features coming together. Looks good to push to main!

File: .vscode/launch.json
Comment (helenliuf) [inline .vscode/launch.json:1]: may not be needed, can rmv later

File: .vscode/settings.json
Comment (helenliuf) [inline .vscode/settings.json:1]: may not be needed, can rmv later

File: team/front-end/source/components/Components.md
Comment (helenliuf) [inline team/front-end/source/components/Components.md:1]: empty file needed?

**Review by jadatwo — APPROVED** [review]
reviewed files, looks good to merge to main.

**Review by helenliuf — APPROVED** [review]
looks good to merge with m3 changes!! some files can be removed later i think

**Comment by aliyabedeen** [comment]
reviewed files, all the code looks clean and proper looks good to merge to main!

**Pull request merged**